### PR TITLE
Merge release/17.6 into develop (wp 17.6-rc-2)

### PR DESCRIPTION
**Pull request by AliSoftware**
The new beta 17.6-rc-2 for WordPress app contains the #14858 fix.

It should also have Google login working now.

### CODE REVIEW

**Comment by AliSoftware** [comment]
Gutenberg got an update in the release branch since I opened this PR, and we plan to do a rc-3 on Monday to include is, so (1) this explains the merge conflict and (2) it could be ok to wait for rc-3 to land before merging it to develop to avoid having to merge twice?

**Review by jkmassel — APPROVED** [review]

**Comment by jkmassel** [comment]
This looks ok code-wise, but CircleCI is grumpy about something in the config – let's get that fixed prior to merging (also we'll probably need a merge branch since `build.gradle` is in conflict.

**Comment by AliSoftware** [comment]
Closing in favor of https://github.com/wordpress-mobile/WordPress-Android/pull/14881

**Pull request closed without merging**
